### ISSUE TITLE
Research and propose a way to add Susi as a contact/friend to messenger services to enable chat communication 

### ISSUE THREAD

**Issue by mariobehling** [loklak/asksusi_messenger#1]
The idea is to use the existing messenger infrastructure of other services and enable people to talk to Susi through their existing chat client. If Susi could be a contact in Telegram or a friend in Facebook people could talk to her directly. We need to understand what options and APIs there are to implement this.
* [ ] Telegram
* [ ] Facebook
* [ ] WeChat
* [ ] Whatsapp
* [ ] Line
* [ ] Viber
* [ ] Google Hangout

**Comment by shivenmian** [comment]
@mariobehling Updated this list.

**Comment by Achint08** [comment]
@mariobehling  There is no text service API available for Google hangout. There is API available for Google talk instead. But rarely someone uses Google Talk. What to do now? Even Whatsapp doesn't have a public API.  Kindly update the list.

**Comment by Achint08** [comment]
@mariobehling Asksusi is not working on facebook.

**Comment by shivenmian** [comment]
@mariobehling @Achint08 Yes, WhatsApp should not be in the list. They ban bots very quickly.

For hangouts, I can see some libraries up there though which you can use.

**Comment by fatimarafiqui** [comment]
@mariobehling @shivenmian Can I start working on gitter bot?

**Comment by shivenmian** [comment]
Yes. Please make a separate repository and work on it. When you are done, drop a message at https://gitter.im/loklak/loklak, and if it is good, we'll add it to the FOSSASIA organisation's repo. :)

**Comment by mariobehling** [comment]
@fatimarafiqui Great to see your interest. Please make a pull request to https://github.com/fossasia/susi_wechatbot Cheers.

**Comment by sujayvenaik** [comment]
I have previously worked on AI Chat bots and its development. And have a fair idea.
We can certainly integrate the bot at various spaces such as Facebook, Slack and even Skype. 
The product i worked on was configured to Fb messesnger making use of web hooks. 
Even made use of Nginx Server for load balancing and other purposes.

Thus, i am interested to work on this.
Skype can also be a platform.
As, i explored it certainly also seems possible on WeChat.
Further how can i start on this ?

**Comment by Abhi2424shek** [comment]
@sujayvenaik pls create a repo on your account and for a platform, if it works we could fork it!

**Comment by AliAyub007** [comment]
@mariobehling as it is done now and not needed should I close this issue?
